mock-identity-system/src/main/resources/application-default.properties

```diff
@@ -113,9 +113,9 @@ mosip.esignet.mock.authenticator.ida.otp-channels=email,phone
 
 mosip.mock.ida.kba.default.field-language=eng
 
-
+mosip.esignet.cache.security.secretkey.reference-id=HEALTH_KEY
```

Comment:
why do we need a new key? We can use the same "mosip.kernel.keymgr.hsm.healthkey.ref-id" @Md-Humair-KK